.github/workflows/task-test.yml

```diff
@@ -86,7 +90,10 @@ jobs:
             tests/pytests/requirements.*
       - name: Setup specific
         working-directory: setup/.install
-        run: ./install_script.sh ${{ steps.mode.outputs.mode }}
+        run: |
+          ./install_script.sh ${{ steps.mode.outputs.mode }}
+          ./install_llvm.sh ${{ steps.mode.outputs.mode }} ${{ env.LLVM_VERSION }}
```

Comment:
If the mode is nothing the second parameter will become the first. Either switch the order (assuming we always have an LLVM version) or use input flags and parser